compiler/msgs.nim

```diff
@@ -566,10 +577,31 @@ proc raiseRecoverableError*(msg: string) {.noinline, noreturn.} =
 
 proc sourceLine*(i: TLineInfo): Rope
 
+let
```

Comment:
@Araq I would, even actually I *did* but it doesn't work. Apparently there is a bug in Nim constant folding that makes ```-``` operator to completely have no effect in this particular case.